### PR TITLE
Update pt-br

### PR DESCRIPTION
**Pull request by SamuelBarbosa0042**
From a brazilian developer that loves wath u doing

### CODE REVIEW

File: android/assets/jsons/translations/Brazilian_Portuguese.properties [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # put the english word 'true' behind the '=', otherwise 'false'.
 # Don't translate these words to your language, only put 'true' or 'false'.
  # Requires translation!
-StartWithCapitalLetter = 
+StartWithCapitalLetter true =
```
Comment (xlenstra) [inline android/assets/jsons/translations/Brazilian_Portuguese.properties:12]: :thinking:

Comment (SamuelBarbosa0042) [reply android/assets/jsons/translations/Brazilian_Portuguese.properties:12]: Sorry, that "behind" on the 9º line toss me away, is this way wrong? Should i put it after the "=" sign?

Comment (xlenstra) [reply android/assets/jsons/translations/Brazilian_Portuguese.properties:12]: It should be after the = sign :)

**Comment by SamuelBarbosa0042** [comment]
Oh, well that's what i tougth, but. Sorry for the mistake, won't happen
again.
Btw, great game, Hope i can contribute more

Em ter, 30 de nov de 2021 20:42, Xander Lenstra ***@***.***>
escreveu:

**Pull request merged**